docs/readthedocs/source/doc/LLM/Overview/KeyFeatures/gpu_supports.md

```diff
@@ -1,47 +1,166 @@
+
+         model = model.to('xpu') # Important after obtaining the optimized model
 ```
 
+### Run Optimized Model
+
+You could then inference using the optimized model on Intel GPUs almostly the same as on CPUs. **The only difference is to set `to('xpu')` for input tensors and `.cpu()` for output tensors.**
+
+Continuing with the [example of Llama-2-7b-chat-hf](#load-and-optimize-model), running as follows:
+```python
+import torch
+
+with torch.inference_mode():
+   prompt = 'Q: What is CPU?\nA:'
+   input_ids = tokenizer.encode(prompt, return_tensors="pt").to('xpu') # With .to('xpu') specifically for inference on Intel GPUs
+   output = model.generate(input_ids, max_new_tokens=32)
+   torch.xpu.synchronize()
+   output = output.cpu() # Move to CPU
+   output_str = tokenizer.decode(output[0], skip_special_tokens=True)
+```
+
+```eval_rst
+.. note::
+
+   The initial generation of optimized LLMs on Intel GPUs could be slow. Therefore, it's advisable to perform a **warm-up** run before the actual generation.
+```
+
+
+## QLoRA Finetuning
+
+We also support finetuning LLMs (large language models) using Q-Lora with BigDL-LLM 4bit optimizations on Intel GPUs.
+
 ```eval_rst
 .. note::
 
-   You may apply INT8 optimizations as follows:
+   Currently, only Hugging Face Transformers models are supported running Q-Lora finetuning.
+```
+
+To help you better understand the finetuning process, here we use model [Llama-2-7b-hf](https://huggingface.co/meta-llama/Llama-2-7b-hf) as an example.
 
-   .. code-block:: python
+**Make sure you have prepared environment following instructions [here](../install_gpu.html). First of all, you need to import `intel_extension_for_pytorch` to run on Intel GPUs**:
 
-      model = AutoModelForCausalLM.from_pretrained('/path/to/model/',
-                                                   load_in_low_bit="sym_int8",
-                                                   optimize_model=False)
-      model = model.to('xpu')
+```python
+import intel_extension_for_pytorch as ipex
 ```
 
-After loading the Hugging Face *Transformers* model, you may easily run the optimized model as follows:
+First, load model using `transformers`-style API and **set it to `to('xpu')`** as follows:
+```python
+import intel_extension_for_pytorch as ipex
+from bigdl.llm.transformers import AutoModelForCausalLM
+
+model = AutoModelForCausalLM.from_pretrained("meta-llama/Llama-2-7b-hf",
+                                             load_in_low_bit="nf4",
+                                             optimize_model=False,
+                                             torch_dtype=torch.float16,
+                                             modules_to_not_convert=["lm_head"],)
+model = model.to('xpu')
+```
 
+Then, we have to apply some preprocessing to the model to prepare it for training.
 ```python
-# run the optimized model
-from transformers import AutoTokenizer
+from bigdl.llm.transformers.qlora import prepare_model_for_kbit_training
```

Comment:
Instead of import `prepare_model_for_kbit_training` and `get_peft_model` from `peft`, as we did for regular qlora using bitandbytes and cuda. Here we import them from `bigdl.llm.transformers.qlora` to get a BigDL-LLM compatible peft model.

Reply:
The rest of the code is same as regular LoRA finetuning using "peft"

Reply:
> Instead of import `prepare_model_for_kbit_training` and `get_peft_model` from `peft`, as we did for regular qlora using bitandbytes and cuda. Here we import them from `bigdl.llm.transformers.qlora` to get a BigDL-LLM compatible peft model.

Have added related descriptions.